src/accordion/__tests__/Accordion.test.tsx

```diff
@@ -32,12 +32,6 @@ const AccordionComponent = (props: any) => {
 };
 
 describe("Accordion", () => {
-  it("should render correctly", () => {
-    const { asFragment } = render(<AccordionComponent />);
```

Comment:
pass the baseId={"myid"} so it does not change everytime the component is rendered

Reply:
We need to pass the base id in the state & for each panels.